### PR TITLE
Fix/collection view resizing

### PR DESCRIPTION
**Pull request by richardoti**
Not the best piece of code I wrote, but it works flawlessly.
Before: https://youtu.be/cdGz0pHzVFQ
After: https://youtu.be/eSbhP-tDn5k

### CODE REVIEW

File: Source/ImagePickerController.swift [hunk 1]
```diff
@@ -301,26 +312,25 @@ extension ImagePickerController: ImageGalleryPanGestureDelegate {
...
       galleryView.frame.origin.y = initialFrame.origin.y + translation.y
       galleryView.frame.size.height = initialFrame.height - translation.y
-    }
 
-    if galleryHeight > GestureConstants.minimumHeight {
-      galleryView.collectionViewLayout.invalidateLayout()
-      galleryView.collectionView.frame.size.height = galleryView.frame.size.height - ImageGalleryView.Dimensions.galleryBarHeight
-      galleryView.collectionSize = CGSize(width: galleryView.collectionView.frame.height, height: galleryView.collectionView.frame.height)
+      let value = self.view.frame.width * (scale - 1) / scale
+      self.galleryView.collectionView.contentInset = UIEdgeInsets(top: 0, left: 0, bottom: 0, right:  value)
```
Comment (zenangst) [inline Source/ImagePickerController.swift]: Do we need to reference `self` here?

Comment (richardoti) [inline Source/ImagePickerController.swift]: Nope. :+1:

File: Source/ImagePickerController.swift [hunk 2]
```diff
@@ -301,26 +312,25 @@ extension ImagePickerController: ImageGalleryPanGestureDelegate {
   func panGestureDidChange(translation: CGPoint) {
     let galleryHeight = initialFrame.height - translation.y
```
Comment (zenangst) [inline Source/ImagePickerController.swift]: You could do 
```swift
guard let galleryHeight = initialFrame.height - translation.y where galleryHeight >= GestureConstants.maximumHeight else { return }
```

File: Source/ImagePickerController.swift [hunk 3]
```diff
@@ -301,26 +312,25 @@ extension ImagePickerController: ImageGalleryPanGestureDelegate {
...
+      self.galleryView.collectionView.contentInset = UIEdgeInsets(top: 0, left: 0, bottom: 0, right:  value)
 
-      if galleryHeight < GestureConstants.maximumHeight {
-        let realTranslation = translation.y < -GestureConstants.minimumHeight + ImageGalleryView.Dimensions.galleryBarHeight
-          ? translation.y + GestureConstants.minimumHeight - ImageGalleryView.Dimensions.galleryBarHeight
-          : translation.y
-        galleryView.collectionView.contentOffset = CGPoint(x: initialContentOffset.x - (realTranslation * CGFloat(numberOfCells)), y: 0)
-      }
+    } else {
+
```
Comment (zenangst) [inline Source/ImagePickerController.swift]: You can remove this newline

Comment (richardoti) [inline Source/ImagePickerController.swift]: I added newlines here intentionally to give more clarity. This is a math-heavy code so I decided to format it in in blocks. Try to look at the source, not the diff. Still want to remove?

File: Source/ImagePickerController.swift (hunk 2 above)
Comment (richardoti) [inline Source/ImagePickerController.swift]: I tried this:
screen shot 2015-10-14 at 22 58 28

So I guess, the solution in this PR right now is quite elegant and easy-to-read.
The only part which I can rewrite right now is early return, and rewriting it with `guard` will not make it any simpler.

**Pull request merged**